test/types/records/generic/allGenericsDefaulted-err3.good

```diff
@@ -0,0 +1,3 @@
+allGenericsDefaulted.chpl:43: error: unresolved call 'baz(R(real(64),2))'
+allGenericsDefaulted.chpl:17: note: this candidate did not match: baz(r: R(int, ?rank))
+allGenericsDefaulted.chpl:17: note: because an argument was incompatible
```

Comment:
Clearly not about this PR, but I am not sure if should try to make the error messages in {1,2} to be similar to {3,4}. The latter set seems more vague, and it can be less helpful if the function had many arguments.

Reply:
I don't disagree, but haven't felt sufficiently inspired to file the issue (yet).  Let me know if you think it's important that I do so (vs. waiting for someone to truly be annoyed enough by it to get around to it themselves).

Reply:
Nope. This error message definitely falls into "good enough for now" category for me.